src/Simplex/Messaging/Version.hs

```diff
@@ -7,7 +7,7 @@
 
 module Simplex.Messaging.Version
   ( Version,
-    VersionRange (minVersion, maxVersion),
+    VersionRange (..),
```

Comment:
it was not exported for a reason, there is a safe smart constructor mkVersionRange to use outside of this module. You should not be able to construct VRange outside of this module